### ISSUE TITLE
[FEATURE] Documentação em Inglês

### ISSUE THREAD

**Issue by emmanuelkant** [b2wads/grimorio-ui#116]
**Descrição do problema relativo a essa feature**
Vi que alguns arquivos de documentação estão em inglês, mas não todos. Com a documentação em inglês podemos ganhar mais visualização no exterior. A ferramenta é muito boa e merece ser usada, mas com a barreira da língua, fica tudo mais difícil. O código já está em inglês, ótimo, mas a porta de entrada (documentação inicial do GitHub e por consequência, do NPM) está em português. Talvez isso mude as nossas visitas e downloads mensais.

**Descreva a solução sugerida e o porquê dessa decisão**
Traduzi todos os documentos que estavam em português. Achei interessante ter uma ferramenta, boa como essa, em um nível internacional.

**Conteúdo adicional**
Um ótimo próximo passo é, que os arquivos de contribuição também tenham seu idioma alterado, bem como o nosso idioma na hora que estivermos abrindo uma issue ou um PR. Isso passa mais segurança quando outras pessoas de outros países, ou que tenham um idioma materno diferente do nosso, quiserem contribuir.

**Comment by apfzvd** [comment]
Oi @emmanuelkant, tudo bom?

Olha, ótimos pontos que você levantou! Gerou uma discussão entre o time porque não tínhamos nenhuma definição concreta sobre o idioma do projeto.

Alguns pontos que pensei foram:

- Por mais que adotar o inglês como padrão do projeto gere uma possibilidade de inclusão para contribuidores estrangeiros, isso acontece em detrimento da inclusão dos muitos desenvolvedores brasileiros que não falam inglês ou que tem um inglês mediano/instrumental. Hoje existem muitos outros ui-kits internacionalizados e acredito que possa ser um diferencial do grimorio seguir pela vertente de mater o português como nosso idioma oficial no projeto.

- Caso todos concordem com essa decisão, precisaremos rever nossas documentações para padronizar o idioma para português!

- Essa possível decisão não significa que não possamos ter uma documentação localizada, também. Na pasta `.github` poderíamos ter uma pasta .translations e lá pastas com as versões dos arquivos de documentação localizados. (Ex.: `/en/README.md`, `/en/docs/git-flow.md`).

Que acham? @dehmirandac2 @daltonmatos @lucasdomi

**Comment by daltonmatos** [comment]
O que penso é que fazendo em Português temos também a oportunidade de produzir conteúdo de qualidade na nossa língua, que é algo que encontramos pouco no nosso dia a dia.

**Comment by dehmirandac2** [comment]
> Oi @emmanuelkant, tudo bom?
> 
> Olha, ótimos pontos que você levantou! Gerou uma discussão entre o time porque não tínhamos nenhuma definição concreta sobre o idioma do projeto.
> 
> Alguns pontos que pensei foram:
> 
> * Por mais que adotar o inglês como padrão do projeto gere uma possibilidade de inclusão para contribuidores estrangeiros, isso acontece em detrimento da inclusão dos muitos desenvolvedores brasileiros que não falam inglês ou que tem um inglês mediano/instrumental. Hoje existem muitos outros ui-kits internacionalizados e acredito que possa ser um diferencial do grimorio seguir pela vertente de mater o português como nosso idioma oficial no projeto.
> * Caso todos concordem com essa decisão, precisaremos rever nossas documentações para padronizar o idioma para português!
> * Essa possível decisão não significa que não possamos ter uma documentação localizada, também. Na pasta `.github` poderíamos ter uma pasta .translations e lá pastas com as versões dos arquivos de documentação localizados. (Ex.: `/en/README.md`, `/en/docs/git-flow.md`).
> 
> Que acham? @dehmirandac2 @daltonmatos @lucasdomi

Concordo! Acho que vai ser legal criarmos material na nossa língua, pra aumentar a inclusão de quem não é fluente em língua inglesa. Mas vejo com bons olhos termos tradução para inglês (e quem sabe pra outras línguas), assim como a @apfzvd citou.

De qualquer maneira, muito legal você ter contribuído com nosso projeto, @emmanuelkant 
Toda sugestão é super bem-vinda! =)

**Comment by emmanuelkant** [comment]
Olá pessoal, muito obrigado pela resposta. Vejo que a criação de conteúdo de qualidade em português é bem escassa hoje em dia, e eu acho muito legal que nossa língua seja priorizada, algo que já vejo muito hoje em dia, em filmes, na cultura gringa, em vídeo games e outros canais que antes, eram exclusivamente na língua inglesa. Eu gostei muito da ideia que a @apfzvd levantou que é colocar uma documentação regional, dessa forma não perdemos nada, apenas ganhamos.

Para ser sincero quando eu acesso o projeto, o que eu, ou qualquer um que o acesse, vê é: código todo inglês, storybook em inglês, algumas documentações em inglês e apenas pouco mais de meia duzia de arquivos em português. Até aqui no GitHub usamos expressões em inglês como fix, feat, enchacement...e até mesmo o nome da ferramenta tem....Grimório-**UI**.

Enfim, minha intenção não é, e nem será, mudar a essência do projeto ou desviar o público-alvo ou nada do tipo. Só ajudar mesmo.

**Comment by daltonmatos** [comment]
Isso aí @emmanuelkant. Essa é a ideia. Em relação aos termos em inglês (código e etc) penso que o segredo é a moderação. Em alguns casos não temos como fugir de usar termos em inglês, por exemplo nas palavras reservadas das linguagens e termos que são mais comuns ou que não comprometem o entendimento do todo.

O legal é poder ter a documentação oficial em portugês e isso nos dá a oportunidade de receber bem quem não tem um inglês fluente mas mesmo assim pode ser beneficiado pelo projeto.

**Comment by emmanuelkant** [comment]
Opa @daltonmatos, show de bola. Realmente com a documentação em português ajuda as pessoas a simpatizarem com o projeto logo de cara né. Vou dar uma olhada em outras issues para ver no que mais posso contribuir.

**Comment by daltonmatos** [comment]
Isso aí!! :tada: 

Estamos em busca de uma ferramnta que seja capaz de produzir documentação multi-língua. Seria um fit perfeito para o projeto. Se conhecer alguma não deixa de postar aqui (pode abrir novas issues) para podermos discutir e eventualmente escolher uma.

**Comment by apfzvd** [comment]
> Para ser sincero quando eu acesso o projeto, o que eu, ou qualquer um que o acesse, vê é: código todo inglês, storybook em inglês, algumas documentações em inglês e apenas pouco mais de meia duzia de arquivos em português.

De fato. Como eu disse antes, não tínhamos uma definição concreta de idioma no projeto, o que acabou acarretando nessa mistura de docs em inglês e português. Vamos precisar traduzir grande parte desses artigos para o português! Porém, acredito no equilíbrio que o @daltonmatos mencionou, também.

> Estamos em busca de uma ferramenta que seja capaz de produzir documentação multi-língua.

Procurando, vi essa ferramenta que é mais para localização de aplicações: https://github.com/marketplace/crowdin. Vou dar uma estudada melhor na ferramenta, mas talvez seja uma opção!
